app/web/src/features/communities/events/EventTimeChanger.tsx

```diff
@@ -139,6 +149,10 @@ export default function EventTimeChanger({
               value: timePattern,
             },
             validate: (time) => {
+              if (!touched.startTime) {
+                return true;
+              }
```

Comment:
I'll comment on this but it's the same across the date and time fields, but when this is used in the edit event context, without this, the react-hook-form validation would still prevent the user from updating the event even if they haven't touched this. Not sure if there's a cleaner way to disable the validation temporaily other than like this.